src/panels/config/integrations/integration-panels/dynalite/dynalite-config-panel.ts

```diff
@@ -0,0 +1,279 @@
+				  <paper-listbox
+					slot="dropdown-content"
+					selected=${this._activeIndex}
+                    @iron-select=${this._handleActiveSelection}
+				  >
+					${this._activeOptions.map(
+            (option) =>
+              html`<paper-item .active_config=${option}
+                >${this.localStr("active_" + option)}</paper-item
+              >`
+          )}
+				  </paper-listbox>
+				</ha-paper-dropdown-menu>
+			  </ha-settings-row>
+            </div>
+            <div class="card-content">
+			  <ha-settings-row .narrow=${this.narrow}>
+				<span slot="heading">${this.localStr("auto_discover")}</span>
+				<span slot="description">${this.localStr("auto_discover_long")}</span>
+				<ha-switch
+				  .checked=${this._auto_discover}
+				  @change=${this._handleAutoDiscoverChange}
+				></ha-switch>
+			  </ha-settings-row>
+            </div>
+            <div class="card-content">
+			  <ha-settings-row .narrow=${this.narrow}>
+				<span slot="heading">${this.localStr("poll_timer")}</span>
+				<span slot="description">${this.localStr("poll_timer_long")}</span>
+                <paper-input
+                  class="flex"
+                  .label=${this.localStr("poll_timer")}
+                  name="poll_timer"
+                  type="number"
+                  .value=${this._poll_timer}
+                  @value-changed=${this._handleChange}
+                >
+			  </ha-settings-row>
+            </div>
+            <div class="card-actions">
+              <mwc-button @click=${this._publish}>
+			    ${this.localStr("publish")}
+			  </mwc-button>
+            </div>
+          </ha-card>
+        </div>
+      </ha-app-layout>
+    `;
+  }
+
+  private localStr(item) {
+    return this.hass.localize("ui.panel.config.dynalite." + item);
+  }
+
+  protected async firstUpdated() {
+    const configEntryId = this._getConfigEntry();
+    if (!configEntryId) return;
+    const response = await this.hass.connection.sendMessagePromise({
+      type: "dynalite/get_entry",
+      entry_id: configEntryId,
+    });
+    this.entryData = response.data;
+    this._name = this.entryData.name;
+    this._host = this.entryData.host;
+    this._port = this.entryData.port;
+    if (this.entryData.default) {
+      this._fade = this.entryData.default.fade;
+    }
+    const activeMap = {
+      on: 0,
+      true: 0,
+      init: 1,
+      false: 2,
+      off: 2,
+    };
+    this._activeIndex = activeMap[this.entryData.active];
+    this._active = this._activeOptions[this._activeIndex].config;
+    this._auto_discover = this.entryData.autodiscover;
+    this._poll_timer = this.entryData.polltimer;
+  }
+
+  private _getConfigEntry() {
+    const searchParams = new URLSearchParams(window.location.search);
+    if (!searchParams.has("config_entry")) {
+      return false;
+    }
+    return searchParams.get("config_entry") as string;
+  }
+
+  private _handleChange(ev: PolymerChangedEvent<string>) {
```

Comment:
We shouldn't be using anything Polymer

Reply:
ok. i copied it from the core config panel. any better example to use?

Reply:
Just use Custom event

Reply:
```suggestion
  private _handleChange(ev: CustomEvent) {
```

Reply:
Using the event type of polymer is not bad, as that is the correct type for the element/event you are using.

Reply:
so should i change it back? you guys know better than me

Reply:
Yea go ahead an switch it back but make sure to re import it as well

Reply:
how do i check for missing imports? no clear explanation on how to run the lint etc. for the frontend, so sorry for the newbie questions

Reply:
`yarn run lint`